shell/src/main/java/alluxio/shell/command/MountCommand.java

```diff
@@ -83,6 +86,21 @@ public Options getOptions() {
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
+    if (args.length == 0) {
```

Comment:
can you also update the `public String getUsage()` method, to have a second line to show the usage of no arg in addition to the one with two args